spec/features/cart/checkout_spec.rb

```diff
@@ -36,4 +36,32 @@
       expect(page).to_not have_link("Checkout")
     end
   end
+  describe 'When admin tried to access cart or merchant pages' do
+    it 'there is a 404 error message' do
+      admin = User.create(name: "Lord Farquaad",
+                          address: "123 Castle Lane",
+                          city: "Duloc City",
+                          state: "Duloc State",
+                          zip: 10001,
+                          email: "lord.farquaad@castle.gov",
+                          password: "Password",
+                          role: 2)
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+      visit "/merchant"
+
+      expect(page).to have_content("The page you were looking for doesn't exist (404)")
+    end
+  end
 end
+# ```
```

Comment:
Test is Looking good! Maybe we can take out the comments here. I usually do it at the very end, in a clean-up commit, or an overall branch. Not sure how the group wants to approach this.

Reply:
Oh yeah I missed those, will delete next run around. Thanks!